PSGSuite/Public/Reports/Get-GSActivityReport.ps1

```diff
@@ -60,7 +68,7 @@ function Get-GSActivityReport {
         [String]
         $UserKey = 'all',
         [parameter(Mandatory = $false,Position = 1)]
-        [ValidateSet("Admin","Calendar","Drive","Groups","GPlus","Login","Mobile","Rules","Token")]
+        [ValidateSet("AccessTransparency","Admin","Calendar","Chat","Drive","Gcp","Gplus","Groups","GroupsEnterprise","Jamboard","Login","Meet","Mobile","Rules","Saml","Token","UserAccounts")]
```

Comment:
Make this ValidateSet Dynamic with this code:
```
[parameter(Mandatory = $false,Position = 1)]
[ArgumentCompleter(
    {
        param($Command, $Parameter, $WordToComplete, $CommandAst, $FakeBoundParams)
        $possibleValues = [Google.Apis.Admin.Reports.reports_v1.ActivitiesResource+ListRequest+ApplicationNameEnum].GetEnumNames()
        $possibleValues | Where-Object {$_ -like "$WordToComplete*"}
    }
)]
[ValidateScript({[Google.Apis.Admin.Reports.reports_v1.ActivitiesResource+ListRequest+ApplicationNameEnum].GetEnumnames()})]
[String]
$ApplicationName = "Admin",
```
That will use the list from the enum to validate against, and also use it for argument completion (aka pressing Tab).